draft-ietf-httpbis-priority.md

```diff
@@ -487,7 +489,7 @@ For example, when the client sends an HTTP request with
 priority = u=4, i=?1
 ~~~
 
-and the origin responds with
+and the origin server responds with
```

Comment:
@kazuho here and elsewhere "origin server" disambiguates from HTTP "origin" RFC6454 https://www.rfc-editor.org/rfc/rfc6454.html